drivers/video/mt9m114.c

```diff
@@ -461,12 +503,36 @@ static int mt9m114_get_caps(const struct device *dev, enum video_endpoint_id ep,
 	return 0;
 }
 
+static int mt9m114_set_ctrl(const struct device *dev, unsigned int cid, void *value)
+{
+	int ret = 0;
+
+	switch (cid) {
+	case VIDEO_CID_HFLIP:
+		ret = mt9m114_set_horizontal_mirror(dev, (int)value);
```

Comment:
s/mirror/flip/

Reply:
I really like the `vim` format of your comment!

Reply:
:)
Note that it of course doesn't apply if you end up refactoring the two functions into one

Reply:
Hi @kartben , as you said, this doesn't apply anymore due to refactoring (implemented differently), could you resolve the Change Request to help moving this PR forwards ?